### PR TITLE
Cleanup Ingress guide

### PR DESCRIPTION
**Pull request by amwat**
Use markdown shortcodes to preview files.

/cc @BenTheElder

### CODE REVIEW

**Comment by amwat** [comment]
https://deploy-preview-1164--k8s-kind.netlify.com/docs/user/ingress/

File: site/content/docs/user/ingress.md [hunk 1]
```diff
@@ -47,116 +57,31 @@ Apply the [mandatory ingress-nginx components](https://kubernetes.github.io/ingr
...
-            servicePort: 5678
----
-```
+{{< codefromfile file="static/examples/ingress/usage.yaml" lang="yaml" >}}
 
 Apply the contents
 
 ```shell script
-kubectl apply -f https://kind.sigs.k8s.io/manifests/ingress/nginx/example.yaml
+kubectl apply -f https://kind.sigs.k8s.io/examples/ingress/usage.yaml
```
Comment (BenTheElder) [inline site/content/docs/user/ingress.md]: Do we have a sane way to template the base URL so this works on the previews etc.?

Can be another follow up

File: site/content/docs/user/ingress.md [hunk 2]
```diff
@@ -47,116 +57,31 @@ Apply the [mandatory ingress-nginx components](https://kubernetes.github.io/ingr
...
----
-kind: Service
-apiVersion: v1
-metadata:
-  name: foo-service
-spec:
-  selector:
-    app: foo
-  ports:
-    - port: 5678 # Default port used by the image
```
Comment (BenTheElder) [inline site/content/docs/user/ingress.md]: Nit thought: We should put the comments on their own lines to make it less wide.

**Pull request merged**